pkg/scheduler/framework/plugins/interpodaffinity/filtering.go

```diff
@@ -370,7 +285,7 @@ func (pl *InterPodAffinity) AddPod(ctx context.Context, cycleState *framework.Cy
 	if err != nil {
 		return framework.NewStatus(framework.Error, err.Error())
 	}
-	state.updateWithPod(podToAdd, podToSchedule, nodeInfo.Node(), 1)
+	state.updateWithPod(podToAdd, nodeInfo.Node(), 1)
```

Comment:
Add a TODO to remove `podToSchedule` from the interface signature?

Reply:
we can't because it is not a standard that every plugin will store the pod in their StateCycle.